components/prism-nevod.js

```diff
@@ -0,0 +1,79 @@
+Prism.languages.nevod = {
+	'comment': /(?:\/\/.*)|(?:\/\*[\s\S]*?(?:\*\/|$))/,
+	'string': [
+		{
+			pattern: /"(?:""|[^"])*"(?!")!?\*?/,
+			greedy: true,
+			inside: {
+				'modifiers': /!$|!\*$|\*$/,
+			},
+		},
+		{
+			pattern: /'(?:''|[^'])*'(?!')!?\*?/,
+			greedy: true,
+			inside: {
+				'modifiers': /!$|!\*$|\*$/,
+			},
+		}
+	],
+	'keyword': /@(?:inside|outside|having|search|where)\b/,
+	'require': /@require\b/,
+	'namespace': {
+		pattern: /@namespace\s+[a-zA-Z0-9\-.]+\s*{/,
+		inside: {
+			'keyword': /@namespace\b/,
+			'name': /\b[a-zA-Z0-9\-.]+/
+		},
+	},
+	'pattern': {
+		pattern: /(?:@pattern\s+)?#?[a-zA-Z0-9\-.]+\s*(?:(?:[(]\s*(?:(?:~\s*)?[a-zA-Z0-9\-.]+\s*(?:,\s*(?:(?:~\s*)?[a-zA-Z0-9\-.]*))*)[)])\s*)?=/,
+		inside: {
+			'keyword': /@pattern\b/,
+			'name': {
+				pattern: /^(\s+)?#?[a-zA-Z0-9\-.]+/,
+				lookbehind: true
+			},
+			'equal': /=/,
+			'attributes': {
+				pattern: /\(.*\)/,
+				inside: {
+					'attr-name': /[a-zA-Z0-9\-.]+/,
+					'attr-hidden-mark': /~/,
+					'attr-punctuation': /[,]/,
+					'attr-bracket': /[()]/,
+				},
+			},
+		},
+	},
+	'search': {
+		pattern: /(?:@search\s+|#)[a-zA-Z0-9\-.]+(?:\.\*)?\s*(?=;)/,
```

Comment:
The last `\s*` should be included in the lookahead. Tokens should generally not include leading or trailing spaces.

Reply:
Fixed and improved.